integration-tests/bats/signed.bats

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env bats
+load $BATS_TEST_DIRNAME/helper/common.bash
+
+setup() {
+    setup_common
+}
+
+teardown() {
+    assert_feature_version
+    teardown_common
+}
+
+# init_gpg() {
```

Comment:
Uncomment?

Reply:
No.  I had to move them to go tests for now.  Dustin and I had issues getting gpg to work with bats